stat/distuv/gamma.go

```diff
@@ -219,11 +219,17 @@ func (g Gamma) Rand() float64 {
 		d := a - 1.0/3
```

Comment:
This above comment is out of date now. Please add a reference for the modification you added.

Reply:
The comment is still correct. The cited paper also describes the new approach.

Reply:
Oh, I see. Thanks.